packages/components/src/components/accordion/accordion.stories.ts

```diff
@@ -34,143 +41,43 @@ export const Default = {
 };
 
 /**
- * An accordion item can either be collapsed or open.
+ * Use the attribute `open` to set the state of the accordion.
  */
-export const States = {
-  parameters: { controls: { exclude: 'open' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'open' }
-      },
-      args,
-      constants: { type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' }
-    });
-  }
-};
 
-/**
- * The summary can have multiple lines.
- */
-export const SummaryLength = {
-  parameters: { controls: { exclude: 'summary' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'slot',
-          name: 'summary',
-          values: [
-            { value: '<slot slot="summary">Lorem ipsum.</slot>', title: 'short' },
-            {
-              value:
-                '<slot slot="summary">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</slot>',
-              title: 'long'
-            }
-          ]
-        }
-      },
-      args,
-      constants: [
-        { type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' },
-        { type: 'slot', name: 'summary', value: '<div slot="summary" class="slot slot--text">Test</div>`' }
-      ]
-    });
-  }
+export const Open = {
+  name: 'Open',
+  render: () => html`
+    <div class="grid grid-cols-2 gap-12">
+      <sd-accordion open summary="Open"
+        ><div class="slot slot--border slot--background slot--text h-16">Default slot</div></sd-accordion
+      >
+      <sd-accordion summary="Closed"
+        ><div class="slot slot--border slot--background slot--text h-16">Default slot</div></sd-accordion
+      >
+    </div>
+  `
 };
 
 /**
- * Use the expand-icon and collapse-icon slots to change the expand and collapse icons, respectively.
- * To disable the animation, override the rotate property on the summary-icon part as shown below:
+ * Use `summary` to provide the text in the accordion header.
  *
- * ```
- * sd-accordion.custom-icons::part(summary-icon) {
- *   rotate: none;
- * }
- * ```
+ * __Hint:__ The `summary` slot allows you to use custom html in the summary.
```

Comment:
* __Hint:__ The `summary` slot allows you to use custom html.

Reply:
Updated